tests/common/devices/multi_asic.py

```diff
@@ -286,6 +286,18 @@ def is_service_running(self, service_name, docker_name=None):
                 return False
 
         return True
+    
+    def get_asic_index_for_port(self, portname):
```

Comment:
can you reuse this:

https://github.com/Azure/sonic-mgmt/blob/7baf71175fdfd696de48fd5ad58f04eac11c2129/tests/common/devices/multi_asic.py#L290

Reply:
changed in the latest comment